.eslintrc.js

```diff
@@ -42,4 +42,11 @@ module.exports = {
       },
     ],
   },
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.jsx', '.ts', '.tsx', '.android.tsx', '.ios.tsx'],
```

Comment:
fixes most of the no module found issues that we run into when using .android files